### PR TITLE
CallHome OSGI bundle for Carbon kernel 5.2.x products

### PR DESCRIPTION
**Pull request by ctienshi**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -41,16 +41,28 @@
...
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
             <version>1.23</version>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.eclipse.osgi</groupId>
+            <artifactId>org.eclipse.osgi</artifactId>
+            <version>3.11.0.v20160603-1336</version>
+            <optional>true</optional>
```
Comment (kasunsiyambalapitiya) [inline pom.xml]: Shall we extract the versions to properties? WDYT?

File: src/main/java/org/wso2/callhome/internal/CallHomeActivator.java [hunk 2]
```diff
@@ -17,33 +17,32 @@
...
 /**
- * The CallHomeActivator class executes {@link CallHome} and registers it as a {@link ServerStartupObserver}.
+ * The CallHomeActivator class executes {@link CallHome}.
  *
  * @since 1.0.2
  */
 public class CallHomeActivator implements BundleActivator {
-    private static final Log log = LogFactory.getLog(CallHomeActivator.class);
+
+    private static final Logger logger = LoggerFactory.getLogger(CallHomeActivator.class);
```
Comment (kasunsiyambalapitiya) [inline src/main/java/org/wso2/callhome/internal/CallHomeActivator.java]: Shall we use the same identifier for the logger constant in all the classes? which will help to maintain the consistency, Here it is `logger` but in CallHome.java it is `log`. 
WDYT?

File: pom.xml (hunk 1 above)
Comment (ctienshi) [reply pom.xml]: Fixed

File: src/main/java/org/wso2/callhome/internal/CallHomeActivator.java (hunk 2 above)
Comment (ctienshi) [reply src/main/java/org/wso2/callhome/internal/CallHomeActivator.java]: +1. Fixed.

**Pull request merged**